stats/view/view.go

```diff
@@ -28,112 +27,109 @@ import (
 	"go.opencensus.io/tag"
 )
 
-// View allows users to filter and aggregate the recorded events.
-// Each view has to be registered to enable data retrieval. Use New to
-// initiate new views. Unregister views once you don't want to collect any more
-// events.
+// View allows users to aggregate the recorded stats.Measurements.
+// Views need to be passed to the Subscribe function to be before data will be
+// collected and sent to Exporters.
 type View struct {
-	name        string // name of View. Must be unique.
-	description string
+	Name        string // Name of View. Must be unique.
+	Description string // Description is a human-readable description for this view.
 
-	// tagKeys to perform the aggregation on.
-	tagKeys []tag.Key
+	// GroupByTags are the tag keys describing the grouping of this view.
+	// A single Row will be produced for each combination of associated tag values.
+	GroupByTags []tag.Key
```

Comment:
Too verbose. We've already defined tags as what groups Measurements, we use tag.Keys to group only one way.

```go
TagKeys []tag.Key
```

Reply:
What about `GroupBy`? TagKeys just repeats the type name so doesn't really add anything?

Reply:
The reason  I wanted to emphasize grouping is that actually before we had some incorrect documentation that said that these tags were being used to filter, which is obviously wrong.

Reply:
I like `GroupBy`, cool and more descriptive suggestion!

Reply:
I don't like the use of group by. This is a SQL terminology and not common in other language implementation or in any of the specs or docs. It is more confusing to me than having TagKeys (consistence with other languages) and having godoc explaining what it does.

Reply:
I defer to @rakyll's suggestion.

Reply:
In java they are called "Columns" (in the View class) FTR I am: +0 on TagKeys (just repeats the type) +1 on GroupBy (I like SQL, many people know SQL) and I don't really understand where Columns comes from? I am happy to disagree and commit though so @dinooliva @bogdandrutu what's the suggestion here?